platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/MerkleDb.java

```diff
@@ -529,7 +535,7 @@ public void snapshot(final Path destination, final MerkleDbDataSource dataSource
                             + " during reconnect or ISS reporting. Table name={}",
                     tableName);
         }
-        final MerkleDb targetDb = getInstance(destination);
+        final MerkleDb targetDb = getInstance(destination, configuration);
```

Comment:
Ah, this is why `Configuration` is needed instead of just `MerkleDbConfig`...

Reply:
Yup